src/helpers/authHelper.js

```diff
@@ -28,7 +28,7 @@ const comparePassword = (password, hashPassword) => bcrypt.compareSync(password,
    * @returns {String} - jwt token
    */
 const generateToken = (payload) => {
-  const token = jwt.sign({ ...payload }, secret, { expiresIn: '7 days' });
+  const token = jwt.sign({ ...payload }, secret, { expiresIn: '60s' });
```

Comment:
why are we reducing the expiry date here

Reply:
Thanks for this comment. This was an error, I actually used it for testing. I will change it right away. Thanks again.